Makefile

```
@@ -0,0 +1,78 @@
+	rm -rf ./bin
+	rm -rf $(API_DIR)
+	go clean
+	@echo "Clean complete"
+
+# テスト実行
+test:
+	@echo "Running tests..."
+	go test -v ./...
+
+# Protocol BuffersからGoコードを生成
+proto:
```

checkmake (0.2.2)

[warning] Target body for "proto" exceeds allowed length of 5 (9).

(maxbodylength)